clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -363,20 +363,36 @@
                                                                              new Field("max_num_offsets",
                                                                                        INT32,
                                                                                        "Maximum offsets to return."));
+    public static final Schema LIST_OFFSET_REQUEST_PARTITION_V1 = new Schema(new Field("partition",
+                                                                                       INT32,
+                                                                                       "Topic partition id."),
+                                                                             new Field("timestamp", INT64, "Timestamp."));
```

Comment:
Probably useful to add some more detail here on the purpose of the timestamp field. This will help with generating the protocol documentation.